wifinina/wifinina.go

```diff
@@ -1123,6 +1125,23 @@ func (w *wifinina) accept(s sock) sock {
 	return newsock
 }
 
+func (w *wifinina) getRemoteData(s sock) netip.AddrPort {
+
+	if debugging(debugCmd) {
+		fmt.Printf("    [cmdGetRemoteData] sock: %d\r\n", s)
+	}
+
+	sl := make([]string, 2)
```

Comment:
Can we avoid the heap allocation here?

Reply:
I tried this:

```
@@ -1131,8 +1131,8 @@ func (w *wifinina) getRemoteData(s sock) netip.AddrPort {
                fmt.Printf("    [cmdGetRemoteData] sock: %d\r\n", s)
        }
 
-       sl := make([]string, 2)
-       l := w.reqRspStr1(cmdGetRemoteData, uint8(s), sl)
+       var sl [2]string
+       l := w.reqRspStr1(cmdGetRemoteData, uint8(s), sl[:])
        if l != 2 {
                w.faultf("getRemoteData wanted l=2, got l=%d", l)
                return netip.AddrPort{}
```

And then compared before and after using -print-allocs=.

No difference.

Reply:
Just noticed it was a slice of string not byte, and also we are already doing this same thing in `getIP()`